shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/impl/MySQLDDLVisitor.java

```diff
@@ -166,7 +166,7 @@ public ASTNode visitRenameTableSpecification(final RenameTableSpecificationConte
     @SuppressWarnings("unchecked")
     @Override
     public ASTNode visitCreateTable(final CreateTableContext ctx) {
-        CreateTableStatement result = new CreateTableStatement((SimpleTableSegment) visit(ctx.tableName()));
+        CreateTableStatement result = new CreateTableStatement((SimpleTableSegment) visit(ctx.tableName()), null != ctx.notExistClause_().EXISTS());
```

Comment:
Do you think `null != ctx.notExistClause_()` is enough?

Reply:
No,  it will return an Empty notExistClause*Context, it's EXISTS() method will return null

Reply:
Have you ever gave it a try? Since the following `null != ctx.createDefinitionClause()` uses the same method. Is there any difference?

Reply:
Yes, I tested and debug it in my local env.